app/model/commands/PublishAtomCommand.scala

```diff
@@ -141,11 +147,41 @@ case class PublishAtomCommand(id: String, override val stores: DataStores, youTu
     }
   }
 
+  private def parseDescriptionForYoutube(description: String): String = {
```

Comment:
Maybe the method could show more intent so, `removeHtmlTagsForYouTubeDescription`. That might help us in the future to remember why we do this?